webapp/src/components/tables/existingConfigTable.tsx

```diff
@@ -0,0 +1,189 @@
+import React, {useState} from 'react';
+
+import {FormGroup, Table, ButtonGroup, Button, OverlayTrigger, Tooltip} from 'react-bootstrap';
+
+import ActionModal from '../modals/actionModal';
+import DeleteModal from '../modals/deleteModal';
+import ConfigModal from '../modals/configModal';
+
+import {DeleteSvg, EditSvg, ViewSvg} from '../svgIcons/svg';
+
+import './styles.scss';
+
+type Props = {
+    onChange: (config: Configs[]) => void;
+    value: Configs[];
+}
+
+const ExistingConfigTable = ({value, onChange}: Props) => {
+    const [isviewVisible, setIsViewVisible] = useState(false);
+    const [isdeleteVisible, setIsDeleteVisible] = useState(false);
+    const [iseditVisible, setIsEditVisible] = useState(false);
+    const [isaddVisible, setIsAddVisible] = useState(false);
+
+    const [configIndex, setConfigIndex] = useState(0);
+
+    const handleView = (index: number) => {
+        setConfigIndex(index);
+        setIsViewVisible(true);
+    };
+
+    const handleDelete = (index: number) => {
+        setConfigIndex(index);
+        setIsDeleteVisible(true);
+    };
+
+    const handleEdit = (index: number) => {
+        setConfigIndex(index);
+        setIsEditVisible(true);
+    };
+
+    const handleAdd = () => {
+        setIsAddVisible(true);
+    };
```

Comment:
```suggestion
    const handleAdd = () =>  setIsAddVisible(true);
```
fix all these

for arrow function there is no need for explicit return if there is only statement in the function body